### DISCUSSION TITLE
Protect items dropped on death has delay when going /back

### DISCUSSION THREAD

**Discussion by codingvincee** [GriefPrevention/GriefPrevention#743] [Q&A] [answered]
When a player dies and uses /back to get to the position where they died, the items need 10seconds before they get unlocked. Is there a specific setting or reason for this?

**Comment by RoboMWM** [comment, marked as answer]
Sounds more like the PvP spawn protection/item pickup protection. It's configurable, because if you pickup an item after spawning, you're no longer protected from PvP in worlds where PvP is enabled.
